src/drizzleReactHooks.js

```diff
@@ -0,0 +1,203 @@
+import React, {
+  createContext,
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useRef,
+  useState
+} from 'react'
+import PropTypes from 'prop-types'
+import debounce from 'debounce'
+import shallowequal from 'shallowequal'
+
+const Context = createContext()
+export const useDrizzle = () => useContext(Context)
+export const useDrizzleState = (mapState, args) => {
```

Comment:
Not exactly sure what `mapState` means here. Maybe add a comment or have a more descriptive variable name?